webhook.go

```diff
@@ -194,7 +195,12 @@ func main() {
 		fmt.Fprint(w, "OK")
 	})
 
-	r.HandleFunc(hooksURL, hookHandler)
+	if *httpMethods == "" {
+		r.HandleFunc(hooksURL, hookHandler)
+	} else {
+		allowed := strings.Split(*httpMethods, ",")
+		r.HandleFunc(hooksURL, hookHandler).Methods(allowed...)
+	}
 
 	if !*secure {
 		log.Printf("serving hooks on http://%s:%d%s", *ip, *port, hooksURL)
```

Comment:
It would be nice if we extended this log statement with global allowed methods as well. I.e. "serving hooks on http://blabla/hooks/{id} for GET, POST methods.". Something along those lines.

Reply:
Done.